index.html

```diff
@@ -0,0 +1,480 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Technical Document</title>
+    <link rel="stylesheet" href="./styles.css"
```

Comment:
Please great than sign to close style link element

Reply:
- [ ] Kindly indicate the reason for the suggested change. In this case, it would be because it might lead to errors when running the HTML file.